docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -227,7 +254,7 @@ public class User extends PanacheEntity {
      * Adds a new user to the database
      * @param username the username
      * @param password the unencrypted password (it will be encrypted with bcrypt)
-     * @param role the comma-separated roles
+          * @param role the comma-separated roles
```

Comment:
extra whitespace?

Reply:
Fixed. Thanks for flagging @ebullient . Must've elbowed the space bar! :mechanical_arm: